src/app/user/user.model.ts

```diff
@@ -0,0 +1,14 @@
+import { getModelForClass, Prop, Ref } from "@typegoose/typegoose";
+import { House } from "../house/house.model";
+
+export class User {
+  id: string;
+
+  @Prop()
+  name: string;
+
+  @Prop({ ref: "House" })
+  houses: Ref<House>[];
```

Comment:
Here I created the references to the houses on the user

Reply:
What happens if you just did
```js
houses?: House[]
```
only putting the ? to say it is nullable.

Reply:
You would not be able to assign a string array value when creating a user document

Reply:
it would work fine for reading but would have type issues assigning to the property